### DISCUSSION TITLE
聊天

### DISCUSSION THREAD

**Discussion by chai666123** [YimMenu/YimMenu#3284] [Q&A]
能不能增加一个聊天框支持Ctrl+V粘贴？

**Comment by chai666123** [comment]
说一下使用场景，我是中国用户，虽然支持翻译功能，我能看懂他们的语言，但是和他们交流我还需要用翻译软件翻译以后才能和他们交流，可是游戏不支持粘贴功能，以至于翻译出来的文本我需要一个一个敲出来

**Reply by ghost** [reply]
> 说一下使用场景，我是中国用户，虽然支持翻译功能，我能看懂他们的语言，但是和他们交流我还需要用翻译软件翻译以后才能和他们交流，可是游戏不支持粘贴功能，以至于翻译出来的文本我需要一个一个敲出来

But seriously, in the Network/Chat menu tab you can insert any text.  maybe you should switch the keyboard layout to English so that the key combination works

**Reply by chai666123** [reply]
您说的对，其实我很早就看到了这个功能，但是我一直觉得这个功能没什么用，我想要粘贴的时候却没有想到可以在这里可以发送，是我愚蠢了，感谢您的提醒

**Comment by ghost** [comment]
dog cat yum yum
